### PR TITLE
support anthropicBaseURL , openaiAPIKey, anthropicAPIKey cli params

### PR DESCRIPTION
**Pull request by GargantuaX**

### CODE REVIEW

**Comment by ezynda3** [comment]
First off, thanks so much for the PR! It looks good. One small thing though, could you remove the non-English comments in the code or translate them to English? After that, I will merge this.

Cheers!

File: cmd/root.go [hunk 1]
```diff
@@ -26,11 +26,14 @@ import (
...
-	configFile    string
-	messageWindow int
-	modelFlag     string // New flag for model selection
-	openaiBaseURL string // Base URL for OpenAI API
+	renderer         *glamour.TermRenderer
+	configFile       string
+	messageWindow    int
+	modelFlag        string // New flag for model selection
+	openaiBaseURL    string // Base URL for OpenAI API
+	anthropicBaseURL string // 新增 anthropicBaseURL 变量
```
Comment (ezynda3) [inline cmd/root.go:34]: Can we remove these comments?

File: cmd/root.go [hunk 2]
```diff
@@ -79,8 +82,13 @@ func init() {
 	// Add debug flag
 	rootCmd.PersistentFlags().
 		BoolVar(&debugMode, "debug", false, "enable debug logging")
-	rootCmd.PersistentFlags().
-		StringVar(&openaiBaseURL, "openai-url", "", "base URL for OpenAI API (defaults to api.openai.com)")
+
+	// Fix: 修改格式，确保每个标志都正确注册
```
Comment (ezynda3) [inline cmd/root.go:86]: Can we remove this comment?

**Review by ezynda3 — CHANGES_REQUESTED** [review]

**Comment by GargantuaX** [comment]
Done

**Pull request merged**